### DISCUSSION TITLE
Stuck on repeating "Idle interval checking for expired" & "encuum memory usage"

### DISCUSSION THREAD

**Discussion by TrevorKinard** [allquixotic/encuum#16] [Q&A]
Is there a solution around being stuck on "TRACE hyper::client::pool: idle interval checking for expired" & "INFO encuum: *** encuum memory usage:"?

I have a subforum that will not complete its pull and is repeating the lines above. 
I tried using the session_id that gets created from running without one, but this did not do anything meaningful.
I have also attempted to use the individual IDs of the threads in place of the "subforum_ids" but that appears to not be supported.

**Comment by allquixotic** [comment]
Hi, I'm not aware of this being a known issue. In my testing, when it's done, it exits cleanly. 

I'm assuming that it can successfully save at least part of the forum to SQLite, right?

Can you identify anything in common about the forum threads/posts that it can't save? Are they announcements, sticky, hidden, or anything other than regular posts?

How many threads/posts does it fail to save? How many does it successfully grab?

Since I haven't experienced this, the more detail I can get, the better I can help.

**Comment by allquixotic** [comment]
Moved to issue https://github.com/allquixotic/encuum/issues/17 - please put all further discussion about this in the issue.
